cmd/noms/noms_sync.go

```diff
@@ -92,8 +93,10 @@ func runSync(args []string) int {
 	if last := <-lastProgressCh; last.DoneCount > 0 {
 		status.Printf("Done - Synced %s in %s (%s/s)", humanize.Bytes(last.DoneBytes), since(start), bytesPerSec(last, start))
 		status.Done()
+	} else if !sinkExists {
+		fmt.Println(args[1], "created.")
```

Comment:
I think maybe something a little more explanatory (and celebratory?)

All chunks already exist at destination! Created new dataset %s.